packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -457,43 +456,17 @@ export class FilesService {
         return this.create(fileInput);
     }
 
-    async copyFilesToScope({ user, fileIds, inboxFolderId }: { user: CurrentUser; fileIds: string[]; inboxFolderId: string }) {
```

Comment:
This could be considered breaking. Alternative would be to keep `user` for compatibility's sake but it does nothing

Reply:
I doubt any application is using this method directly